### DISCUSSION TITLE
Pi-hole

### DISCUSSION THREAD

**Discussion by sn0opy** [selfhst/icons#13] [New Icon Request]
https://pi-hole.net/

Brand guide: https://pi-hole.net/trademark-rules-and-brand-guidelines/
SVG logo available here: https://github.com/pi-hole/web/blob/master/img/logo.svg

**Comment by shollyethan** [comment]
Pi-hole already exists in the collection - is there anything I'm missing?

**Reply by sn0opy** [reply]
Must've missed it. I was searching for "pihole" eventhough I properly requested "pi-hole"...
